talk/expert/cpp20concepts.tex

```diff
@@ -0,0 +1,194 @@
+\subsection[tmpl]{C++20 Requirements and Concepts}
+
+\begin{frame}[fragile]
+  \frametitlecpp[20]{C++20 Requirements and Concepts}
+  \begin{block}{Motivation}
+    Generic programming is made of variable, function and class templates
+    which can be instantiated with different types. It is frequent
+    to instantiate them with \textbf{unsuited types}, and the resulting
+    compilation errors are generally very long and hardly understandable.
+    As a last resort, the template authors are providing \textbf{documentation}
+    about the relevant parameters, and practice some tricky
+    \textbf{template meta-programmation}. C++20 finally brings simpler ways to
+    define constraints on template parameters !.
```

Comment:
Too much text to be readable by the students. We need something structured, with a couple of head lines (e.g. "template difficulties", "Tricky meta-programmation", "The C++20 solution") and 1-3 bullets each time.
And I would put "Motivations" as the title of this slide

Reply:
I tried a bullet list.